src/app/games/games-page/games-page.component.ts

```diff
@@ -0,0 +1,66 @@
+import {Component, OnInit} from '@angular/core';
+import {GamesFacade} from 'src/app/store/facades/games.facade';
+import {startWith, take, tap} from 'rxjs/operators';
+import {AuthFacade} from '../../store/facades/auth.facade';
+
+@Component({
+  selector: 'app-games-page',
+  templateUrl: './games-page.component.html',
+  styleUrls: ['./games-page.component.scss']
+})
+export class GamesPageComponent implements OnInit {
+
+  games$ = this.gamesFacade.userGames$.pipe(startWith([
+      {
+        gameId: 'gameId',
+        dbGameId: 'dbGameID',
+        name: 'CSGO',
+        imageId: 'imgurl/lol',
+        tags: [],
+      }
+    ]),
+    tap((val) => console.log(val)));
+
+  private gamesList: {
+    gameId: string; dbGameId: string; imageId: string; name: string;
+    tags: { tagId: string; name: string; slug: string }[]
+  }[];
+
+  constructor(private readonly gamesFacade: GamesFacade,
+              private readonly userFacade: AuthFacade) {
+  }
+
+  ngOnInit() {
+    this.userFacade.authenticatedUser$.pipe(take(1)).subscribe((user) => {
+      this.gamesFacade.loadUserGames(user.email);
+    });
+    this.gamesList = [{
```

Comment:
[suggestion] move the test-data initialization into a separate method for better readability